### ISSUE TITLE
Add Spotify like button support

### ISSUE THREAD

**Issue by ahmedsalikhov** [dsafa/audio-band#231]
Hello. Thank you for this app.
It'd be really nice if app has a button for like/unlike songs.

**Comment by sorsted** [comment]
This is the _single_ feature I am looking for, and the lack of it is the very feature that will make me install Audio Band. Seriously, please: The Spotify API already includes this feature, so adding it should be qutie simple, right?

**Comment by randallflagg34** [comment]
This enhacement could be really nice!

**Comment by ariccb** [comment]
Yeah i would love this functionality! IF ITS POSSIBLE, adding a button to add current song to a playlist would be incredible too

**Comment by benerogerio** [comment]
This is the missing feature! it would be awesome!

**Comment by zebau** [comment]
Yes, adding current song to liked songs or to a specific playlist would be amazing!

**Comment by mpaustin1993** [comment]
Would love to have this feature!

**Comment by FoxcroftN** [comment]
Looking forward to this feature. Has anyone got something to work regarding the like functionality?

**Comment by svr333** [comment]
not yet but Im planning on adding it right after I finish the volume button Im working on (which is about to be done)
ETA would be before 2021 is over, hopefully

https://github.com/AudioBand/AudioBand if you're interested

**Comment by svr333** [comment]
This feature has been requested a lot and we've finally released a version where it is working.
Checkout the latest release here: https://github.com/AudioBand/AudioBand/releases

@sorsted you may now install AudioBand good sir
![image](https://user-images.githubusercontent.com/35664724/183263932-085c182a-da3d-4ec4-a97d-0d0ae19fa56c.png)

**Comment by ariccb** [comment]
Now, is this something that possibly works on windows 11??

**Comment by sorsted** [comment]
@svr333 Excellent! It works perfectly!

There are some other weird bugs, though. When I click on the song cover, it changes profile. And if I create a new profile, it doesn’t always work unless I also restart Windows Explorer.

**Comment by svr333** [comment]
Thanks for the reports, Ill take a look at them. Theres been a few bug reports that werent uncovered during the testing period :/

**Comment by svr333** [comment]
@ariccb Yes it does work on Windows 11, read the install guide here: https://audioband.github.io/AudioBand/audioband/Windows11.html
